src/Redux/Store.js

```diff
@@ -1,6 +1,8 @@
-import { createStore, applyMiddleware, compose } from "redux";
+import { applyMiddleware, compose, createStore } from "redux";
 import thunk from "redux-thunk";
+
 import rootReducer from "./reducers";
+import sandboxState from '../sandbox/sandbox_state';
```

Comment:
double quotes for imports.. you could try `npm run lint:fix' for the common issues.

Reply:
Same as [comment](https://github.com/singnet/refactored-dapp/pull/110#discussion_r309244829)